services/validate.js

```diff
@@ -19,12 +17,19 @@ fs.readdirSync(baseDir)
   })
   .forEach(function (fileName) {
     try {
-      validator.addSchema(fileName, require(path.join(baseDir, fileName)))
+      validator.addSchema(JSON.parse(path.join(baseDir, fileName)), fileName)
```

Comment:
You're right, tv4 was being too lenient with the format and allowing non-URIs to pass. That's fixed in https://github.com/interledgerjs/five-bells-ledger/pull/379. The tests still don't pass, though.